### DISCUSSION TITLE
Pulseq system information

### DISCUSSION THREAD

**Discussion by byronphy** [pulseq/pulseq#67] [Q&A]
Hi all,

Thank you very much for the great work making sequence design simpler.

I am trying to run the FID using pulseq in VE12U Terra system. 

(1) I wonder where I can find the system parameters of the scanner below? Are they necessary?

`'maxGrad',mr.convert(40,'mT/m'),...   % Default: 40 mT/m
        'maxSlew',mr.convert(170,'T/m/s'),...	% Default: 170 mT/m/ms
        'riseTime',[],...
        'rfDeadTime',0,...
        'rfRingdownTime',0,...
        'adcDeadTime',0,...
        'adcRasterTime',100e-9,...
        'rfRasterTime',1e-6,...
        'gradRasterTime',10e-6,...
        'blockDurationRaster',10e-6,...
        'gamma',42576000,...
        'B0',1.5...`

(copied from opt.m)

(2) We can run the gre_example.seq from the C2P folder on the scanner. But the fid.seq generated by  \ISMRM-Virtual-Meeting--November-15-17-2023-main\tutorials\day1_basic\sequence\s01_FID.m seems to fail to run on the scanner. When we try to run the fid.seq on the scanner, the left down corner of the host screen shows "Scanning... estimating time..." and last for a long time without stopping. I was wondering if there are some issues about the codes `system = mr.opts('rfRingdownTime', 20e-6, 'rfDeadTime', 100e-6, 'adcDeadTime', 20e-6) ;` in s01_FID.m. 

(It seems that the default B0 value is 1.5 T.)

Thank you!

**Comment by m-a-x-i-m-z** [comment]
Hi! 

yes, the system parameters are important to specify before you start creating any Pulseq objects. The system parameters you specify in Pulseq should not strictly match the true limits of the actual MR system you will be using later. The parameters you specify should be "within the limits" of what the actual MR scanner can do. Parameters  'rfRingdownTime' and 'adcDeadTime' are actually tricky ones, as the majority of Siemens MR scanners can do 10 us, but for some scanners and coils 20 us may be necessary. That is why we set it to 20 us in the tutorial. 

Similarly, B0 specified in the system properties is irrelevant as long as you do not explicitly use system frequency, e.g. to calculate fat suppression RF pulses. 

The problem you describe on your 7T system is not known to us and has not been reported before. Does any other example sequence run (e.g. the built-in sequence or the gre_example.seq delivered with the Pulseq Interpreter)? 

So, because you've hit a quite unusual condition, you may need to debug it a bit on the scanner. A way to get started is to run the LogViewer before starting the sequence and then check for suspicious messages.

Best regards, 
Maxim
